package/yast2-storage.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Jun  9 11:41:38 CEST 2015 - shundhammer@suse.de
+
+- No snapper config for LVM storage proposal (bsc#922308)
```

Comment:
The change log entry is a bit unclear, was no snapper config fixed or do we want to have no snapper config for LVM? (It's clear from the bug description, but here the wording is a bit misleading, at least for me...)